conf/application.conf

```diff
@@ -182,7 +182,7 @@ feature-switch {
 #  In the Staging Environment, this will be flipped to true
 #  Therefore changing the state of FS(s) will be easier during demos
 feature-switches {
-  read-from-mongo = false
+  read-from-mongo = true
```

Comment:
I would prefer this flag to stag false in local as its in PROD, plus there will be quite a lot of failing UT/ITs as far as I know, we might need to wait for EMA team to complete they work around replacing current auth infrastructure with a new approach, before fixing these tests;